### DISCUSSION TITLE
How does bird net 'Choose' a bird in a clip?

### DISCUSSION THREAD

**Discussion by tomwhite101** [mcguirepr89/BirdNET-Pi#483] [General]
Hi, 
I am just trying to increase my understanding of how the system works here, if anyone is able to enlighten?

I was reviewing recordings today and this specto clearly shows two birds singing. A Wood Pigeon (in the >1kHz range), which is recorded and something else (a Blue Tit I think, in the 3-9kHz range) in the same 6 second clip.

How does the system deal with this, does it consider both and then record the one with greatest confidence? Am I wrong in my assumption it doesn't record both, does it or **could it** actually pick out two birds singing at the same time?

https://user-images.githubusercontent.com/80456902/183612629-a8e0d4f9-07af-40f4-9e38-e8d31088f86e.mp4

very best,
Tom

**Comment by CaiusX** [comment]
Hi Tom, yes it would report both if both are above the cutoff confidence.  

Best 

AP
